### DISCUSSION TITLE
Improve price calculations by combining fixed and multipler to pricing

### DISCUSSION THREAD

**Discussion by zelbanna** [UtilitechAS/amsreader-firmware#663] [Ideas]
Hi,

Since there are two (should be at least) mutually exclusive fields called fixed price and multiplier my suggestion is to add the behavior so that price would be p (cost/kWh) = fixed + multiplier * regional_hourly_cost. 
- if there is no fixed price defined -> treat 'fixed' as 0 which is the same behavior as today
- if there is no region specified -> the price would be fixed which is the same behavior as today ( I think/hope)

Using above formula would better cover the end customer price which usually includes fixed fees and taxes coupled to actual energy consumption and allow for VAT to be added to the regional price which is unknown beforehand

**Comment by gskjold** [comment]
Thank you for your input, I will keep this in mind as I update this part of the code. There are currently a couple of issues with suggestions for the price. Planning on making it a priority soon :)
